### PR TITLE
ticket 10 sort by functionality

### PR DESCRIPTION
**Pull request by Daniel17971**

### CODE REVIEW

**Review by HeatherEver — CHANGES_REQUESTED** [review]
Nice work Daniel! Works well and it looks neat but there are a couple of things missing: 
- a user should also be able to sort by ascending and descending too so please add this to your app
- this sort by should also be available when the reviews are filtered by their topics ⭐ 
Please make these changes and submit another PR

File: src/components/AllReviews.jsx [hunk 1]
```diff
@@ -39,6 +42,24 @@ const AllReview = () => {
...
+  };
+  const handleChangeOrder = (event) => {
+    if (event.target.value === "asc") {
+      setAllReviews((currentReviews) => {
+        return [...currentReviews].reverse();
+      });
+    }
+    if (event.target.value === "desc") {
+      setAllReviews((currentReviews) => {
+        return [...currentReviews].reverse();
```
Comment (Pipe-Smoking-Rabbit) [inline src/components/AllReviews.jsx:58]: This is certainly one way of changing the sort order, but your backend should hopefully have the capacity to do this in it's optional queries. It's more efficient for processes like this to be handled  by the backend wherever possible

Comment (Pipe-Smoking-Rabbit) [inline src/components/AllReviews.jsx:58]: This is certainly one way of changing the sort order, but your backend should hopefully have the capacity to do this in it's optional queries. It's more efficient for processes like this to be handled by the backend wherever possible

**Review by Pipe-Smoking-Rabbit — CHANGES_REQUESTED** [review]
It does now effectively change the sort order as well, I would however advise to do this using a new fetch request making use of the queries on the endpoint. If that is not possible on your backend then that's fine 😃 
One other thing I have noticed regarding the styling is that the side bar covers most of the content when viewing in a mobile view port ( narrowed screen width ), it's not super important for right now but that is something I would advise trying to adjust at some point as we're trying to design mobile first 📱

**Pull request merged**